### ISSUE TITLE
How can I use the xlwings in flask?

### ISSUE THREAD

**Issue by szjss** [xlwings/xlwings#757]
I have a VBA Macro named Proj_Result. I can use it in Spyder by import xlwings. My code is following:
`import xlwings as xw`
`wb = xw.books('Macro_for_Proj_Result.xlsm')`
`pr = wb.macro('Proj_Result')`
`result=pr(ws1, runno1, prod_name1, spcode, result_type, var, r.date.strftime('%Y/%m/%d'))`
`print(result)`

I run the python script and get the correct result. I want use this code in web application so I use flask framestructure. The code is same as in spyder but I get a error on this line:

`wb = xw.books('Macro_for_Proj_Result.xlsm')`
OR
`wb = xw.books(os.path.join(jspt.static_folder, 'Macro_for_Proj_Result.xlsm')) `
And I have put the file Macro_for_Proj_Result.xlsm in jspt.static_folder and open it. My config for jspt.static_folder is : jspt=Blueprint('jspt',name,template_folder='templates',stati‌​c_folder='../static'‌​) 

And the error is following:

```
Traceback (most recent call last):
  File "C:\Anaconda3\lib\site-packages\flask\app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\lib\site-packages\flask\app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\inetpub\wwwroot\lapydev\jspt\views.py", line 428, in getvariable
    getAIOdata(md5_code,left_result,middle_result,right_result)
  File "C:\inetpub\wwwroot\lapydev\jspt\views.py", line 440, in getAIOdata
    getVardata(aioParams,left_result[0][0])
  File "C:\inetpub\wwwroot\lapydev\jspt\views.py", line 446, in getVardata
    wb = xw.books(os.path.join(jspt.static_folder, 'Macro_for_Proj_Result.xlsm'))
  File "C:\Anaconda3\lib\site-packages\xlwings-0.11.4-py3.5.egg\xlwings\main.py", line 49, in __call__
    return self._wrap(impl=self.impl(name_or_index))
  File "C:\Anaconda3\lib\site-packages\xlwings-0.11.4-py3.5.egg\xlwings\main.py", line 2828, in impl
    return apps.active.books.impl
  File "C:\Anaconda3\lib\site-packages\xlwings-0.11.4-py3.5.egg\xlwings\main.py", line 358, in books
    return Books(impl=self.impl.books)
  File "C:\Anaconda3\lib\site-packages\xlwings-0.11.4-py3.5.egg\xlwings\_xlwindows.py", line 374, in books
    return Books(xl=self.xl.Workbooks)
  File "C:\Anaconda3\lib\site-packages\xlwings-0.11.4-py3.5.egg\xlwings\_xlwindows.py", line 302, in xl
    self._xl = get_xl_app_from_hwnd(self._hwnd)
  File "C:\Anaconda3\lib\site-packages\xlwings-0.11.4-py3.5.egg\xlwings\_xlwindows.py", line 218, in get_xl_app_from_hwnd
    ptr = accessible_object_from_window(child_hwnd)
  File "C:\Anaconda3\lib\site-packages\xlwings-0.11.4-py3.5.egg\xlwings\_xlwindows.py", line 189, in accessible_object_from_window
    byref(IDispatch._iid_), byref(ptr))
  File "_ctypes/callproc.c", line 920, in GetResult
OSError: [WinError -2147467259] 未指定的错误
--------------------------------------------------------------------------------
INFO in __init__ [C:/inetpub/wwwroot/lapydev/__init__.py:65]:
hello 500!
--------------------------------------------------------------------------------
```
I use the web server of pycharm. How can I use the xlwings in flask? Thank you very much!

**Comment by QGB** [comment]
_xlwindows.accessible_object_from_window

**Comment by fzumstein** [comment]
This should be resolved since 0.13.0.
